numpy/__init__.py

```diff
@@ -185,8 +185,12 @@ def pkgload(*packages, **options):
     pkgload.__doc__ = PackageLoader.__call__.__doc__
 
     from .testing import Tester
-    test = Tester().test
-    bench = Tester().bench
+    if '.dev0' in __version__:
+        test = Tester(raise_warnings="develop").test
+        bench = Tester(raise_warnings="develop").bench
+    else:
+        test = Tester(raise_warnings="release").test
+        bench = Tester(raise_warnings="release").bench
```

Comment:
Minor nitpick, but this could be a tiny bit more DRY as something like:
```
if '.dev0' in __version__:
    raise_warnings = 'develop'
else:
    raise_warnings = 'release'
tester = Tester(raise_warnings=raise_warnings)
test = tester.test
bench = tester.bench
```
Separate instances of `Tester` can still be used to remain DRY if it's necessary, but is it necessary for some reason? (If so a comment would be helpful because it left me wondering.)